RFCs/backend/database/introduction.md

```diff
@@ -3,3 +3,5 @@
 This topic directory concerns the design of the database scheme. 
 
 The description of each table is named `<table_name>_table.md`. 
+
+Without special statements, `models.<field_name>` refers to `django.db.models.<field_name`. 
```

Comment:
it seems to be `django.db.models.<field_name>`

Reply:
Good catch!